### DISCUSSION TITLE
Solution for "CocoaPods could not find compatible versions for pod "GoogleDataTransport""

### DISCUSSION THREAD

**Discussion by asika32764** [capawesome-team/capacitor-mlkit#142] [General]
For someone who meet same problem that shows this error when running: `ionic cap sync ios`

> [capacitor]         [!] CocoaPods could not find compatible versions for pod "GoogleDataTransport":
[capacitor]         In Podfile:
[capacitor]         CapacitorMlkitBarcodeScanning (from `../../node_modules/@capacitor-mlkit/barcode-scanning`) was resolved to 5.4.0, which depends on
[capacitor]         GoogleMLKit/BarcodeScanning (= 4.0.0) was resolved to 4.0.0, which depends on
[capacitor]         GoogleMLKit/MLKitCore (= 4.0.0) was resolved to 4.0.0, which depends on
[capacitor]         MLKitCommon (~> 9.0.0) was resolved to 9.0.0, which depends on
[capacitor]         GoogleDataTransport (~> 9.0)
[capacitor]         
[capacitor]         None of your spec sources contain a spec satisfying the dependency: `GoogleDataTransport (~> 9.0)`.
[capacitor]         
[capacitor]         You have either:
[capacitor]         * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
[capacitor]         * mistyped the name or version.
[capacitor]         * not added the source repo that hosts the Podspec to your Podfile.
[capacitor]         
[ERROR] An error occurred while running subprocess capacitor.

My solution is to run this commands:

```
cd /path/to/ionic-project/ios/App
arch -x86_64 pod install --repo-update
```

Then run again:

```
ionic cap sync ios
```

**Comment by robingenz** [comment]
Thanks for sharing.
